qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -1266,6 +1273,164 @@
+        "field": "author_affiliations",
+        "contentType": "string",
+        "ellipsis": true
+      },
+      {
+        "name": "Journal",
+        "field": "journal",
+        "contentType": "string"
+      },
+      {
+        "name": "Publication URL",
+        "field": "publication_url",
+        "contentType": "string"
+      },
+      {
+        "name": "Condition",
+        "field": "condition",
+        "contentType": "string"
+      },
+      {
+        "name": "Websites",
+        "field": "websites",
+        "contentType": "string"
+      }
+    ],
+    "studyPreviewField": {
+      "name": "Abstract",
+      "field": "abstract",
+      "contentType": "string",
+      "includeName": true,
+      "includeIfNotAvailable": false
+    },
+    "studyPageFields": {
+      "header": {
+        "field": "title"
+      },
+      "fieldsToShow": [{
+          "groupName": "Publication Identifiers",
+          "includeName": false,
+          "fields": [
+          {
+            "name": "Authors",
+            "field": "authors",
+            "contentType": "string"
+          },
+          {
+            "name": "Affiliations",
+            "field": "author_affiliations",
+            "contentType": "string"
+          },
+          {
+            "name": "Journal",
+            "field": "journal",
+            "contentType": "string"
+          },
+          {
+            "name": "Publication URL",
+            "field": "publication_url",
+            "contentType": "string"
+          },
+          {
+            "name": "Data Availability Date",
+            "field": "data_availability_date",
+            "contentType": "string"
+          },
+          {
+            "name": "Brief Description of Data Available",
+            "field": "data_description",
+            "contentType": "string"
+          },
+          {
+            "name": "Sponsors/Acknowledgements",
+            "field": "sponsors",
+            "contentType": "string"
+          },
+          {
+            "name": "Study Number/Codes",
+            "field": "study_codes",
+            "contentType": "string"
+          },
+          {
+            "name": "Condition",
+            "field": "condition",
+            "contentType": "string"
+          },
+          {
+            "name": "Study Type",
+            "field": "study_type",
+            "contentType": "string"
+          },
+          {
+            "name": "Websites",
+            "field": "websites",
+            "contentType": "string"
+          }
+          ]
+        },
+        {
+          "fields": [{
+            "name": "Abstract",
+            "field": "abstract",
+            "contentType": "paragraphs",
+            "includeName": false,
+            "includeIfNotAvailable": false
+          }]
+        }
+      ]
+    },
+    "minimalFieldMapping": {
+      "tagsListFieldName": "tags",
+      "authzField": "auth_resource_path",
+      "uid": "_publication_id"
```

Comment:
what is `_publication_id`? is that what you chose as a unique ID to submit to the MDS?

Reply:
We can change it later if PRC has different requirements.